Lib/posixpath.py

```diff
@@ -538,45 +538,45 @@ def relpath(path, start=None):
         raise
 
 
-# Return the longest common sub-path of the sequence of paths given as input.
+# Return the longest common sub-path of the iterable of paths given as input.
```

Comment:
This is a bugfix that will presumably be backported, so we shouldn't change the types accepted by `commonpath()`.

Reply:
It isn't part of this bugfix, that was changed here: #114709, but the occurences of sequence weren't updated.